DscConfigurationData/Public/Resolve-DSCConfigurationProperty.ps1

```diff
@@ -136,7 +136,7 @@ function Resolve-DscConfigurationProperty
             throw 'Failed to resolve ConfigurationData.  Please confirm that $ConfigurationData is property set in a scope above this Resolve-DscConfigurationProperty or passed to Resolve-DscConfigurationProperty via the ConfigurationData parameter.'
         }
     }
-
+    $value = @()
```

Comment:
This was weird. Don't know if I changed anything but Resolve-DSCConfigurationProperty started erroring since value wasn't initialised when the count is checked. Initializing the variable fixed the problem.